# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/performance/tests/test_performance_observer.js
# Commit: 5248abfed275
# Full Hash: 5248abfed275b2938ed64770bec58b3ff68d624b
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-12 21:56:33
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=mstange,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27008
# ==============================================================================

diff -r e17500911824 -r 5248abfed275 dom/performance/tests/test_performance_observer.js
--- a/dom/performance/tests/test_performance_observer.js	Fri Apr 12 15:49:22 2019 +0000
+++ b/dom/performance/tests/test_performance_observer.js	Fri Apr 12 16:29:48 2019 +0000
@@ -12,11 +12,11 @@
   var observer = new PerformanceObserver(() => {
   });
 
-  assert_throws({name: "TypeError"}, function() {
+  assert_throws({name: "SyntaxError"}, function() {
     observer.observe();
   }, "observe() should throw TypeError exception if no option specified.");
 
-  assert_throws({name: "TypeError"}, function() {
+  assert_throws({name: "SyntaxError"}, function() {
     observer.observe({ unsupportedAttribute: "unsupported" });
   }, "obsrve() should throw TypeError exception if the option has no 'entryTypes' attribute.");
 